layers/ii-yaml/packages.el

```diff
@@ -70,4 +70,6 @@ Each entry is either:
 (defun ii-yaml/init-flycheck-swagger-tools ()
 (use-package flycheck-swagger-tools)
 )
+;; TODO needs better logic to only apply this to kubernetes manifests
+(puthash :kubernetes "/*yaml" (symbol-value 'lsp-yaml-schemas))
```

Comment:
This allows for k8s manifest editing ala https://octetz.com/docs/2020/2020-01-06-vim-k8s-yaml-support/

Though we don't yet have support for specifying the exact swagger.json definiton https://github.com/redhat-developer/yaml-language-server/issues/211